packages/dds/tree/src/feature-libraries/modular-schema/genericFieldKindFormat.ts

```diff
@@ -13,10 +24,10 @@ import { Static, TAnySchema, TSchema, Type } from "@sinclair/typebox";
  * `Static<ReturnType<typeof EncodedGenericChange<Schema>>>`
  */
 class Wrapper<T extends TSchema> {
-	public encodedGenericChange(e: T) {
+	public encodedGenericChange(e: T): TTuple<[TNumber, T]> {
```

Comment:
I think this one and the one below fall under the "intentionally derived" category. Even after seeing and reading the return type here, I still have to click into the inner thing to know what it is - so, I have to drill down either way.

Reply:
I'll probably do a global disable for this module then.